content/rs/administering/active-active.md

```diff
@@ -39,15 +39,11 @@ Before configuring a CRDB, you must:
 ## CRDB Current Limitations
 
 1. RS is limited to five Participating Clusters or CRDB Instances per CRDB.
-1. An existing database cannot modified to be a CRDB.
-    To move existing data to a CRDB you must create a new CRDB and migrate your data.
+1. To move data from an existing database to a CRDB you must create a new CRDB and migrate the data.
```

Comment:
The new limitation is written in a way it isn't clear about transforming an existing DB to become a CRDB.
We need to say something like:
An existing database cannot be modified to be a CRDB.
To do so, you must create a new CRDB and migrate your data.